README.md

```diff
@@ -90,7 +90,8 @@ Note to install the software below we recommend the following package managers:
   - Install the required gems by running `bundle install` in the project directory.
   - Copy `config/database.yml.example` to `config/database.yml` and edit `database.yml` to point to your database.
   - Copy `config/thinking_sphinx.yml.example` to `thinking_sphinx.yml` and adjust any settings (usually not necessary).
-  - Copy `config/initializers/local_config.rb.example` to `config/initializers/local_config.rb` and adjust any settings.
+  - Copy `config/secrets.yml.example` to `config/secrets.yml` and run `rake secrets` to generate keys for development and test environmets
+  - Copy `config/initializers/secret_token.rb.example` to `config/initializers/secret_token.rb` and adjust any settings.
```

Comment:
Can we just put secret token and secret key base in local_config.rb.example? That's then two less files that need to be copied on setup.